CromIAM/src/test/scala/cromiam/webservice/SubmissionSupportSpec.scala

```diff
@@ -4,14 +4,17 @@ import akka.event.{LoggingAdapter, NoLogging}
 import akka.http.scaladsl.model._
 import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken, RawHeader}
 import akka.http.scaladsl.server.AuthorizationFailedRejection
-import akka.http.scaladsl.testkit.ScalatestRouteTest
+import akka.http.scaladsl.testkit.{RouteTestTimeout, ScalatestRouteTest}
 import org.scalatest.{FlatSpec, Matchers}
+import scala.concurrent.duration._
 
 class SubmissionSupportSpec extends FlatSpec with Matchers with ScalatestRouteTest with SubmissionSupport {
   override val cromwellClient = new MockCromwellClient()
   override val samClient = new MockSamClient()
   override val log: LoggingAdapter = NoLogging
 
+  implicit val routeTestTimeout = new RouteTestTimeout(10.seconds)
```

Comment:
This isn't amenable to the scaling magic we've been using successfully elsewhere?

Reply:
If this is problematically slow, this should be `10.seconds.dilated` available via the magic of `import akka.testkit._`.